cloudflared.rb

```diff
@@ -1,4 +1,5 @@
 class Cloudflared < Formula
+  disable! date: "2024-01-10", because: "moved to homebrew-core repository"
```

Comment:
Let's favor deprecate for cloudflared since the package is still valid and working. In a year we can remove it.
```suggestion
  deprecate! date: "2024-01-10", because: "moved to homebrew-core repository"
  disable! date: "2025-01-10", because: "moved to homebrew-core repository"
```